bindings/sampling-heap-profiler.cc

```diff
@@ -81,13 +81,26 @@ NAN_METHOD(GetAllocationProfile) {
   info.GetReturnValue().Set(TranslateAllocationProfile(root));
 }
 
+NAN_METHOD(GetAllocationProfileProto) {
+  if (info.Length() != 1 || !info[0]->IsNumber()) {
+    return Nan::ThrowTypeError("First argument type must be Integer.");
```

Comment:
The condition also checks the number of arguments but the message does not reflect that.

Reply:
done.